GOVERNANCE.md

```diff
@@ -16,6 +16,9 @@ will be automatically removed unless a
 the other project maintainers agrees to extend the period to be greater than 3
 months.
 
+If a maintainer violates our [code of conduct](CODE_OF_CONDUCT.md), a [super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) vote will be
+performed by the other project maintainers to determine the future of the maintainer.
```

Comment:
I want to use a vote so that the "incident" can be discussed and we don't just throw people out.

Maybe that should be clarified:
```suggestion
If a maintainer violates our [code of conduct](CODE_OF_CONDUCT.md), all maintainers will discuss the incident allowing the maintainer to give background and other maintainers will cast a vote. If the vote results in a  [super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote), then the maintainer needs to step down and leave the maintainer board.
```

Reply:
I like the suggestion

Reply:
Nit: there seems to be an extra space in `a  [super-majority]`